packages/client/src/options.ts

```diff
@@ -56,13 +56,15 @@ export const DEFAULT_ConnectionOptions: Partial<ConnectionOptions> = {
  */
 export interface WriteOptions {
   /** Precision to use in writes for timestamp. default ns */
-  precision: WritePrecision
+  precision?: WritePrecision
   /** HTTP headers that will be sent with every write request */
   headers?: {[key: string]: string}
   /** When specified, write bodies larger than the threshold are gzipped  */
   gzipThreshold?: number
   /** InfluxDB Enterprise write consistency as explained in https://docs.influxdata.com/enterprise_influxdb/v1.9/concepts/clustering/#write-consistency */
   consistency?: 'any' | 'one' | 'quorum' | 'all'
+  /** default tags */
+  defaultTags?: {[key: string]: string}
```

Comment:
Please include an example demonstrating how to initialize the client with default tags in the documentation strings for either `WriteOptions` or `InfluxDBClient`.